ui/app/AppLayouts/Onboarding2/LoginWithKeycardFlow.qml

```diff
@@ -84,20 +90,23 @@ SQUtils.QObject {
         id: keycardEnterPinPage
 
         KeycardEnterPinPage {
-            tryToSetPinFunction: root.tryToSetPinFunction
+            id: keycardEnterPinPage
+            keycardState: root.keycardState
+            authorizeFunction: root.authorizeFunction
             remainingAttempts: root.remainingAttempts
             unlockUsingSeedphrase: true
 
-            onKeycardPinEntered: (pin) => {
-                Backpressure.debounce(root, root.keycardPinInfoPageDelay, () => {
-                    root.keycardPinEntered(pin)
-                    root.finished()
```

Comment:
When are we emitting `finished` here now?

Reply:
I changed this since your last review. It is now called below `onExportKeysDone`